agent/osv_output_handler.py

```diff
@@ -216,7 +228,7 @@ def parse_vulnerabilities_osv_api(
         if old_risk is not None and new_risk is not None and old_risk > new_risk:
             highest_risk_vuln_info["risk"] = risk
             highest_risk_vuln_info["cvss_v3_vector"] = _get_cvss_v3_vector(
-                vulnerability.get("severity")
+                vulnerability.get("severity", [])
```

Comment:
There is a check for `severity_data is not None` in `_get_cvss_v3_vector` can you remove it